### PR TITLE
Introduce a tool for comparing benchmark results

### PR DESCRIPTION
**Pull request by ligurio**
Question about license: https://github.com/google/benchmark/issues/1891

### CODE REVIEW

File: perf/lua/benchmark.lua [hunk 1]
```diff
@@ -80,6 +80,12 @@ local function add_result(bench, name, data)
         cpu_time = data.cpu_time,
         iterations = data.items,
         items_per_second = items_per_second,
+        run_name = name,
+        run_type = 'iteration',
+        repetitions = 1,
+        repetition_index = 1,
+        threads = 1,
+        time_unit = 'ns',
```
Comment (Buristan) [inline perf/lua/benchmark.lua:88]: Should it be `'s'` instead?

**Review by Buristan — APPROVED** [review]
Hi, Sergey!
Thanks for the patchset!
I will proceed with the review per-patch below.

---

**[PATCH 1/3] perf/lua: fix benchmark output format**

LGTM, with a minor comment below.

**Review by Buristan — APPROVED** [review]
**[PATCH 3/3] perf: remove script compare.lua**

LGTM.

**Comment by ligurio** [comment]
Added a license to Python files and updated commit message.

File: perf/lua/benchmark.lua (hunk 1 above)
Comment (ligurio) [reply perf/lua/benchmark.lua:88]: Fixed:

```diff
--- a/perf/lua/benchmark.lua
+++ b/perf/lua/benchmark.lua
@@ -85,7 +85,7 @@ local function add_result(bench, name, data)
         repetitions = 1,
         repetition_index = 1,
         threads = 1,
-        time_unit = 'ns',
+        time_unit = 's',
     }
     table.insert(bench.results, result)
     return result
```

**Review by mkokryashkin — APPROVED** [review]
Thanks for the patchset.
Per patch review is below:

[perf: remove script compare.lua](https://github.com/tarantool/tarantool/pull/10875/commits/469628f979657f38322fb6c71f3213529f5cbce1)
LGTM

[perf: add a tool for comparing benchmark results](https://github.com/tarantool/tarantool/pull/10875/commits/a6884cde3698bec97aedbf3266fedc42926f1b49)
LGTM, except for a few comments regarding the commit message:

> The patch add a `compare.py`, versatile benchmark output compare

Typo: s/a `compare.py`, versatile/`compare.py`, a versatile/

> tool, see [1] and [2]. It is useful to compare benchmark results
with each other. Python source code was added without functional
changes, typo was fixed in a file

Typo: s/typo was fixed in a file/a typo was fixed in the file/

>`perf/tools/compare/gbench/util.py` and a text of the license was

Typo: s/a text/the text/

> added, see [3]. The tool has optional dependencies scipy and
numpy, both can be installed if necessary.


[perf/lua: fix benchmark output format](https://github.com/tarantool/tarantool/pull/10875/commits/631ea1ee5f423c0e2f5d66f9244500d2ad6dc15b)
LGTM

**Pull request merged**